rasa/nlu/training_data/loading.py

```diff
@@ -123,8 +123,15 @@ def _load(filename: Text, language: Optional[Text] = "en") -> Optional["Training
         return None
 
 
-def _guess_format(filename: Text) -> Text:
-    """Applies heuristics to guess the data format of a file."""
+def guess_format(filename: Text) -> Text:
+    """Applies heuristics to guess the data format of a file.
+
+    Args:
+        filename: Text type with the file name.
```

Comment:
I think we shouldn't put the name in the docstring since it's already in the code. Having it twice means also double the maintenance effort. I think something like `filename: file whose type should be guessed` would be better

Reply:
Done :D